app/test_split_integer.py

```diff
@@ -2,20 +2,22 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    total = split_integer(6, 2)
+    assert sum(total) == 6
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(6, 2) == [3, 3]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    result = split_integer(17, 4)
+    assert len(result) == 4 and result[0] == 4
```

Comment:
This test should check if the value is equal to return part when number_of_parts = 1, for example split_integer(8, 1).